src/kirby/components/modal/modal-wrapper/modal-wrapper.component.html

```diff
@@ -1,59 +1,33 @@
-<div
-  class="modal-wrapper"
-  #modalWrapper
-  tabindex="-1"
-  (window:focus)="onFocusChange()"
-  (window:focusout)="onFocusChange()"
->
-  <div class="modal-header">
-    <div *ngIf="config.flavor === 'drawer'">
+<ion-header>
+  <ion-toolbar>
+    <ion-title [class.drawer]="config.flavor === 'drawer'">{{ config.title }}</ion-title>
+    <ion-buttons slot="start" *ngIf="config.flavor === 'drawer'">
       <ng-container *ngTemplateOutlet="closeButton; context: { icon: 'arrow-down' }"></ng-container>
-    </div>
-    <span
-      class="title"
-      [class.text-center]="config.flavor === 'drawer'"
-      [class.text-left]="config.flavor === 'modal'"
-    >
-      {{ config.title }}
-    </span>
-    <div *ngIf="config.flavor === 'modal'">
+    </ion-buttons>
+    <ion-buttons slot="end" *ngIf="config.flavor === 'modal'">
       <ng-container *ngTemplateOutlet="closeButton; context: { icon: 'close' }"></ng-container>
-    </div>
-    <div *ngIf="config.flavor === 'drawer' && config.drawerSupplementaryAction">
+    </ion-buttons>
+    <ion-buttons slot="end" *ngIf="config.flavor === 'drawer' && config.drawerSupplementaryAction">
```

Comment:
Is config.falvor unnecessary as it says **drawer**SupplementaryAction?
```suggestion
    <ion-buttons slot="end" *ngIf="config.drawerSupplementaryAction">
```

Reply:
The API says: "drawerSupplementaryAction: [...] Note that this is only available on modals with a 'drawer' flavor" - so I'll keep the check :)